### PR TITLE
Update the Payment Methods columns

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request updates the Payment Methods columns

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

https://docs.magento.com/user-guide/configuration/sales/braintree.html
https://docs.magento.com/user-guide/configuration/sales/paypal-express-checkout.html

### CODE REVIEW

File: src/configuration/sales/braintree.md [hunk 1]
```diff
@@ -37,11 +37,11 @@ _Braintree Advanced Settings_
 |Field|[Scope]({% link configuration/scope.md %})|Description|
 |--- |--- |--- |
 |Vault Title|Website|A descriptive title for your reference that identifies the vault where your customer card information is stored.|
-|Merchant Account ID|Website|The Merchant ID that is to be associated with Braintree transactions from this website. If left blank, the default merchant account from your Braintree account is used.|
+|Merchant Account ID|Store View|The Merchant ID that is to be associated with Braintree transactions from this website. If left blank, the default merchant account from your Braintree account is used.|
```
Comment (leslietilling) [inline src/configuration/sales/braintree.md:40]: This says `website` in my 2.4.3 instance: 

image

**Review by leslietilling — CHANGES_REQUESTED** [review]
Just a fix on the one change.

File: src/configuration/sales/braintree.md (hunk 1 above)
Comment (ZhovnirMaria) [reply src/configuration/sales/braintree.md:40]: @leslietilling Oh thank you, my mistake 😊

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling the Pull Request is successfully imported.

**Pull request merged**